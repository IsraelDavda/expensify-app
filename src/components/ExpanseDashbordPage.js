import React from 'react';
import ExpenseList from './expenseList';
import ExpenseFiltersList from './expenseFiltersList';


const ExpanseDashbordPage = () => (
    <div>
    <ExpenseFiltersList />
    <ExpenseList />
    </div>
);

export default ExpanseDashbordPage;