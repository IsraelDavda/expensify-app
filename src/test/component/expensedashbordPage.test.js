import React from 'react';
import { shallow } from 'enzyme';
import ExpanseDashbordPage from '../../components/ExpanseDashbordPage';

test('should render ExpenseDashbordPage', () => {
    const wrapper = shallow(<ExpanseDashbordPage />);
    expect(wrapper).toMatchSnapshot();
})