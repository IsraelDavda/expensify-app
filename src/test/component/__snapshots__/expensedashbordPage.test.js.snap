// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashbordPage 1`] = `
<div>
  <Connect(ExpenseFiltersList) />
  <Connect(ExpenseList) />
</div>
`;
